runner/Makefile

```diff
@@ -1,9 +1,10 @@
-NAME ?= summerwind/actions-runner
-DIND_RUNNER_NAME ?= ${NAME}-dind
+DOCKER_USER ?= summerwind
+RUNNER_NAME ?= ${DOCKER_USER}/actions-runner
+DIND_RUNNER_NAME ?= ${DOCKER_USER}/actions-runner-dind
```

Comment:
This breaks the instructions in CONTRIBUTING.md
> - Run `NAME=$DOCKER_USER/actions-runner-controller VERSION=canary make docker-build docker-push` for a custom container image build

Reply:
The instructions within the `CONTRIBUTING.md` refer to the Makefile within the root of the repo, these changes are within `runner/`Makefile`